src/writeData/components/PluginCreateConfigurationFooter.tsx

```diff
@@ -1,36 +1,54 @@
 // Libraries
-import React, {FC} from 'react'
+import React, {FC, useEffect, useMemo} from 'react'
 import {connect, ConnectedProps} from 'react-redux'
 
 // Components
 import {Button, ComponentColor, Overlay} from '@influxdata/clockface'
 
 // Actions
-import {
-  addPluginBundleWithPlugins,
-  createOrUpdateTelegrafConfigAsync,
-} from 'src/dataLoaders/actions/dataLoaders'
+import {createOrUpdateTelegrafConfigAsync} from 'src/dataLoaders/actions/dataLoaders'
+import {updateTelegraf} from 'src/telegrafs/actions/thunks'
 
 // Types
-import {BundleName} from 'src/types/dataLoaders'
+import {AppState, ResourceType, Telegraf} from 'src/types'
 import {PluginCreateConfigurationStepProps} from 'src/writeData/components/PluginCreateConfigurationWizard'
 
+// Selectors
+import {getDataLoaders} from 'src/dataLoaders/selectors'
+import {getAll} from 'src/resources/selectors'
+
 type ReduxProps = ConnectedProps<typeof connector>
 type Props = PluginCreateConfigurationStepProps & ReduxProps
 
 const PluginCreateConfigurationFooterComponent: FC<Props> = props => {
   const {
     currentStepIndex,
-    onAddPluginBundle,
     onDecrementCurrentStepIndex,
     onExit,
     onIncrementCurrentStepIndex,
     onSaveTelegrafConfig,
+    onUpdateTelegraf,
     substepIndex,
+    telegrafConfig,
+    pluginConfig,
   } = props
 
+  const shouldTelegrafUpdate = useMemo(() => {
+    if (telegrafConfig) {
+      return true
+    }
+    return false
+  }, [telegrafConfig])
+
+  useEffect(() => {
+    if (telegrafConfig) {
+      const {config} = telegrafConfig
+      onUpdateTelegraf({...telegrafConfig, config: config + pluginConfig})
```

Comment:
```ts
config: config + pluginConfig
```

is this number addition or string concatenation? if it's string concatenation, it **might** be easier to understand like:
```ts
config: `${config}${pluginConfig}`

Reply:
Updated.